src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card-container" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="iMac photo"
+        class="card-photo"
+      >
+      <div class="card-content">
+        <p class="product-name">
```

Comment:
You make new BEM-blocks for card-photo, product-name, product-code, product-rating, etc. You need to create BEM-block for independent components that can be reused. Don't make BEM-blocks for 1 line of text

Reply:
should i change names of classes for small elements such as image to jest 'photo', not 'card-photo'? 
or ive understood wrong?